### PR TITLE
Add bio for eabbeint

### PR DESCRIPTION
**Pull request by eabbeintu**
add a bio for @eabbeintu for issue #17 

@ccampbell-intuit - :heavy_check_mark: it out

### CODE REVIEW

File: docs/eabbeintu.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Bio
+- **Name:** Eli Abbe
```
Comment (johnpanelli) [inline docs/eabbeintu.md:2]: I think Abbe is spelled Abbbbbbbbbeeeeee  :+1:

File: docs/eabbeintu.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- **Title:** SW Engineer
+- **Location:** SF Peninsula
```
Comment (ccampbell-intuit) [inline docs/eabbeintu.md:4]: Hmm, maybe expand "SF"? And include the state?

**Comment by jimt1234** [comment]
here's a thoughtful comment

File: docs/eabbeintu.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Bio
+- **Name:** Eli Abbe
+- **Title:** SW Engineer
```
Comment (ccampbell-intuit) [inline docs/eabbeintu.md:3]: What does "SW" stand for? Southwest? [urbandictionary.com](http://www.urbandictionary.com/define.php?term=SW&defid=3199098) has some interesting definitions as well!

**Comment by dlee01** [comment]
Hey Eli, great job!
Go Quicken!

File: docs/eabbeintu.md (hunk 2 above)
Comment (eabbeintu) [inline docs/eabbeintu.md:4]: I am trying to obfuscate my location

**Pull request merged**